packages/js-dpp/lib/errors/consensus/signature/InvalidIdentityPublicKeySecurityLevelError.js

```diff
@@ -0,0 +1,26 @@
+const AbstractSignatureError = require('./AbstractSignatureError');
```

Comment:
It must be implemented in `IdentityUpdateTransition#getKeySecurityLevelRequirement`